sdk/identity/identity/test/manual/nodeTestSilent.js

```diff
@@ -35,7 +35,7 @@ const credential1 = new identity.InteractiveBrowserCredential({
     // TODO: Perhaps make the account parameter part of the tokenCachePersistenceOptions?
```

Comment:
Should this TODO be cleaned up before we ship?

Reply:
CC @sadasant